test/beetle.py

```
@@ -0,0 +1,131 @@
+    filtered_point = lfilter(b_band, a_band, filtered)
+    return filtered_point[0]
+
+def calculate_focus_level(eeg_data, sampling_rate=500):
+    window = np.hanning(len(eeg_data))  # Apply a Hanning window
+    eeg_data_windowed = eeg_data * window
+    fft_data = np.abs(np.fft.fft(eeg_data_windowed))[:len(eeg_data_windowed) // 2]
+    fft_data /= len(eeg_data_windowed)
+    freqs = np.fft.fftfreq(len(eeg_data_windowed), d=1 / sampling_rate)[:len(eeg_data_windowed) // 2]
+    
+    # Compute power in different bands
+    delta_power = math.sqrt(np.sum((fft_data[(freqs >= 0.5) & (freqs <= 4)]) ** 2))
```

Ruff (0.8.2)

Local variable `delta_power` is assigned to but never used

Remove assignment to unused variable `delta_power`

(F841)